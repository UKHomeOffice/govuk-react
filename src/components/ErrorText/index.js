// https://github.com/alphagov/govuk-frontend/tree/master/src/components

import glamorous from "glamorous";
import { ERROR_COLOUR } from "govuk-colours";
import { FONT_SIZE, MEDIA_QUERIES, NTA_LIGHT } from "../../constants/index";

const ErrorText = glamorous.span({
  fontFamily: NTA_LIGHT,
  WebkitFontSmoothing: "antialiased",
  MozOsxFontSmoothing: "grayscale",
  fontWeight: 700,
  textTransform: "none",
  fontSize: FONT_SIZE.SIZE_16,
  lineHeight: "20px",
  color: `${ERROR_COLOUR}`,
  paddingTop: "4px",
  paddingBottom: "2px",
  [MEDIA_QUERIES.LARGESCREEN]: {
    fontSize: FONT_SIZE.SIZE_19,
    lineHeight: "25px"
  }
});

export default ErrorText;
