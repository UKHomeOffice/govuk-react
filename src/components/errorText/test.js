import React from "react";
import ReactDOM from "react-dom";
import ErrorText from "./";

describe("ErrorText", () => {
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<ErrorText>Example</ErrorText>, div);
  });
});
