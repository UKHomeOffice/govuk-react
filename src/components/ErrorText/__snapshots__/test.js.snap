// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`enzyme.mount 1`] = `
<glamorous(span)>
  <span
    className="css-110ebyw"
  >
    example
  </span>
</glamorous(span)>
`;

exports[`enzyme.render 1`] = `
<span
  class="css-110ebyw"
>
  example
</span>
`;

exports[`enzyme.shallow 1`] = `
<span
  className="css-110ebyw"
>
  example
</span>
`;
